<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="numberLbl">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="activeBox">
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="animBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="speedBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="flashBox">
     <property name="text">
      <string>Flash</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="borderBox">
     <property name="text">
      <string>Border</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="bmpBox">
     <property name="text">
      <string>Bitmap</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacker">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="textEdit">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="fontBtn">
         <property name="text">
          <string>Font...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Î”y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="offsetBox">
         <property name="minimum">
          <number>-100</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="bmpLoadBtn">
         <property name="text">
          <string>Load...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>439</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>activeBox</tabstop>
  <tabstop>animBox</tabstop>
  <tabstop>speedBox</tabstop>
  <tabstop>flashBox</tabstop>
  <tabstop>borderBox</tabstop>
  <tabstop>bmpBox</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>fontBtn</tabstop>
  <tabstop>offsetBox</tabstop>
  <tabstop>bmpLoadBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>speedBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>animBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flashBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>borderBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bmpBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Form</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offsetBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>997</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changed()</slot>
 </slots>
</ui>
